src/arch/ppc-opcode.h

```diff
@@ -0,0 +1,23 @@
+#ifndef __OPCODES_H
```

Comment:
maybe we can move this file into `src/common`, where `crc32c_ppc_asm.S` resides. and it is `crc32c_ppc_asm.S` who is including this header file.